### PR TITLE
Update runtime to 22.08

### PR DESCRIPTION
**Pull request by orowith2os**

### CODE REVIEW

**Comment by orowith2os** [comment]
Something to do with libtool: 
```
libtool: Version mismatch error.  This is libtool 2.4.6, but the
libtool: definition of this LT_INIT comes from an older release.
libtool: You should recreate aclocal.m4 with macros from libtool 2.4.6
libtool: and run autoconf again.
```

**Comment by hfiguiere** [comment]
just run `autoreconf -vfi` in a `shell` source.

File: org.supertux.SuperTux-Milestone1.json [hunk 1]
```diff
@@ -28,6 +28,10 @@
                     "type": "git",
                     "url": "https://github.com/SuperTux/supertux.git",
                     "tag": "v0.1.4"
+                },
+                {
+                    "type": "shell",
+                    "commands": [ "autoreconf -vfi" ]
```
Comment (hfiguiere) [inline org.supertux.SuperTux-Milestone1.json:34]:
```suggestion
                    "commands": [ 
                        "libtoolize",
                        "autoreconf -vfi" 
                    ]
```

I'd try this

**Comment by orowith2os** [comment]
Not able to build locally atm, sorry >>

File: org.supertux.SuperTux-Milestone1.json [hunk 2]
```diff
@@ -28,6 +28,13 @@
                     "type": "git",
                     "url": "https://github.com/SuperTux/supertux.git",
                     "tag": "v0.1.4"
+                },
+                {
+                    "type": "shell",
+                    "commands": [ 
+                        "libtoolize",
+                        "autoreconf -vfi"
```
Comment (hfiguiere) [inline org.supertux.SuperTux-Milestone1.json:36]: it's on the wrong module.

Comment (hfiguiere) [reply org.supertux.SuperTux-Milestone1.json:36]: aka, you can remove it and just update the `shared-modules` as it is smpeg that fails to build. It's usually a good practice to do so when upgrading runtime.

Comment (hfiguiere) [reply org.supertux.SuperTux-Milestone1.json:36]: (it builds locally after doing all of this)

Comment (orowith2os) [reply org.supertux.SuperTux-Milestone1.json]: Thank you!

**Pull request merged**